instance/tests/models/test_database_server.py

```diff
@@ -223,6 +223,50 @@ def test_set_field_defaults(self):
         self.assertEqual(custom_mongodb_server.port, 1234)
 
 
+@override_settings(
+    DEFAULT_INSTANCE_MONGO_URL=None,
+    DEFAULT_MONGO_REPLICA_SET_NAME="test_name",
+    DEFAULT_MONGO_REPLICA_SET_USER="test",
+    DEFAULT_MONGO_REPLICA_SET_PASSWORD="test",
+    DEFAULT_MONGO_REPLICA_SET_PRIMARY="test.opencraft.hosting",
+    DEFAULT_MONGO_REPLICA_SET_HOSTS="test.opencraft.hosting,test1.opencraft.hosting,test2.opencraft.hosting"
+)
+class MongoDBReplicaSetManagerTest(TestCase):
+    """
+    Test cases for MongoDBReplicaSetManager.
+    """
+
+    def test__create_default_replica_set(self):
+        """
+        Test that `_create_default` uses default settings to create MongoDB Replica Set and Servers.
+        """
+        MongoDBReplicaSet.objects._create_default()
+
+        self.assertEqual(MongoDBServer.objects.filter(replica_set__isnull=False).count(), 3)
+        self.assertEqual(MongoDBReplicaSet.objects.count(), 1)
+        for server in MongoDBServer.objects.filter(replica_set__isnull=False):
+            self.assertEqual(server.username, "test")
+            self.assertEqual(server.password, "test")
+
+    @override_settings(
+        DEFAULT_MONGO_REPLICA_SET_USER=None
+    )
+    def test_select_random_fails(self):
+        """
+        Test that `select_random` returns DoesNotExist when one of the settings is not configured
+        """
+        with self.assertRaises(MongoDBReplicaSet.DoesNotExist):
+            MongoDBReplicaSet.objects.select_random()
+        self.assertLogs("instance.models.database_server" "ERROR")
```

Comment:
@rocioar Missing comma between "ERROR" and "instance.models.database_server", or am I missing something? :)

Reply:
wow, how did this pass the pylint checks?